Makefile

```diff
@@ -3,47 +3,54 @@
 PREFIX := /usr
 DESTDIR :=
 
-ifndef VERSION
-MAJORVERSION := 8
-MINORVERSION ?= $(shell git rev-list --count master)
-endif
-VERSION := ${MAJORVERSION}.${MINORVERSION}
+
+MAJORVERSION$(VERSION) := 8
+MINORVERSION$(VERSION) != git rev-list --count master
```

Comment:
Removing the ifndef makes this compatible with BSD make (which uses a slightly different syntax). It should also be compatible with any POSIX compliant make system now. This is probably less necessary than the other changes in this file since it's unlikely anyone will be using anything but GNU make on arch.

Reply:
Changing to `!=` instead of the shell expansion means that this is evaluated once, and not every time the variable is used. Probably doesn't matter since it's only used once that I see here. It was just habit.